pkg/store/etcd.go

```diff
@@ -51,6 +51,7 @@ func InitializeEtcd(addrs []string, options *Config) (Store, error) {
 		}
 	}
 
+	s.client.SyncCluster()
```

Comment:
Not sure about doing a periodic `SyncCluster` in a goroutine for now. I saw that on integration tests but a call to `SyncCluster` could empty your `members` list. I'm not sure about the behavior during the lifetime of the etcd cluster (with machines being added and removed). This would be super unfortunate to have the `discovery` and `leader election` down because of a faulty call to `SyncCluster`.

We should test it thoroughly and see if we can improve the sync process (directly in go-etcd maybe)